packages/protocol/test2/TaikoL1.sim.sol

```diff
@@ -44,13 +43,18 @@ contract TaikoL1Simulation is TaikoL1TestBase, FoundryRandom {
 
     function setUp() public override {
         TaikoL1TestBase.setUp();
+        // TODO(daniel): update string key generation using bytes.concat
         _registerAddress(
             string(abi.encodePacked("verifier_", uint16(100))),
             address(new Verifier())
         );
     }
 
-    function testGeneratingManyRandomBlocks() external {
+    // Disabling this test for now. (We have someshting similar
```

Comment:
This test is where we want to simulate the real world scenarios by randomizing as many parameters as possible without many detailed assertions. Then later we can export some stats and analysis the data by visualizing them, that's why CSV data is exported, one line per block.